### PR TITLE
docs: update release.md for generating verifier e2e tests

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

See title. Updates to describe how verifier e2e test gen works after releasing builders.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -207,15 +207,24 @@ End-to-end tests run daily in [github.com/slsa-framework/example-package/.github
 
 ## Update verifier
 
-The next step is to update the verifier's e2e tests. For this, you need to:
+The next step is to update the verifier's GitHub Actions e2e tests. There are GitHub actions Go and generic actions.
 
-1. Generate binaries and provenance for a project, using the [example-package](https://github.com/slsa-framework/example-package) builder. You will need to create provenance via a `workflow_dispatch` from this [workflow](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.go.workflow_dispatch.main.config-noldflags.slsa3.yml) of the e2e test repository, and push a tag of the form vX and vX.Y to trigger this [workflow](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.go.tag.main.config-ldflags-assets.major.slsa3.yml).
+<!-- TODO: Describe GHA generic container e2e tests. -->
```
Comment (ianlewis) [inline RELEASE.md:212]: nit: create an issue and add to the generic container milestone

**Review by ianlewis — APPROVED** [review]
Thanks. This is fine for now but I think @laurentsimon suggested that we add these instructions to the verifier repo and link to it here.

**Review by laurentsimon — APPROVED** [review]
Thanks!

File: RELEASE.md (hunk 1 above)
Comment (asraa) [reply RELEASE.md]: done! linked here: made two issues, one for creation in example-package, and one for docs linked here.

**Comment by asraa** [comment]
> Thanks. This is fine for now but I think @laurentsimon suggested that we add these instructions to the verifier repo and link to it here.

SG: I'll need to update verifier release docs and can do that in that PR

**Pull request merged**